pkgs/data/fonts/libertinus/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "libertinus-${rev}";
+  rev = "v6.4";
+
+  src = fetchFromGitHub {
+    inherit rev;
+    owner  = "khaledhosny";
+    repo   = "libertinus";
+    sha256 = "0acnq4vpplp2s7kdnhncz61diji3wmhca04g27yqpk03ahb40x9g";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/opentype/
+    mkdir -p $out/share/doc/${name}/
+    cp *.otf $out/share/fonts/opentype/
+    cp *.txt $out/share/doc/${name}/
+  '';
+
+  outputHashAlgo = "sha256";
+  outputHashMode = "recursive";
+  outputHash = "1fjk3vmixl1gmjn0cj5jyc4r933mqm9wrb6kn447nisxmkvn379j";
+
+  meta = with stdenv.lib; {
+    description = "Libertinus fonts is a fork of the Linux Libertine and Linux Biolinum fonts that started as an OpenType math companion of the Libertine font family, but grown as a full fork to address some of the bugs in the fonts.";
```

Comment:
The description is too long.

Reply:
Is there a rule of thumb for how long descriptions should be?

Reply:
It should fit on one line of a full screen terminal (e.g. in the `nox` output), so the 100 character limit is reasonable, with most descriptions being under 80 characters.

Reply:
Thanks for the pointer!